### PR TITLE
Front end styles

### PR DESCRIPTION
**Pull request by Samson303**
What works from previous branch was moved here. Landing page and Settings features for Roosevelt. No text standardising that will be done on new branch.

### CODE REVIEW

File: src/components/Navbar/navbar.module.scss [hunk 1]
```diff
@@ -23,6 +23,10 @@
   }
 }
 
+.userIcon {
```
Comment (MathisBullinger) [inline src/components/Navbar/navbar.module.scss]: This class doesn't seem to be used anywhere.

Keep in mind that this is a CSS module and as such, the styles are not global but have to be explicitly bound to an element.

File: src/screens/Settings/calendar.module.scss [hunk 2]
```diff
@@ -13,6 +13,34 @@
...
       overscroll-behavior: none;
     }
+
+    .rbc-today {
+      background-color: #feeef2;
+      opacity: 0.7 !important;
+    }
+    .rbc-toolbar,
+    .rbc-btn-group {
+      & button {
```
Comment (MathisBullinger) [inline src/screens/Settings/calendar.module.scss]: The `&` is unnecessary.

**Review by MathisBullinger — CHANGES_REQUESTED** [review]

File: src/screens/Settings/calendar.module.scss (hunk 2 above)
Comment (Samson303) [reply src/screens/Settings/calendar.module.scss]: Yes you are right. It was just what I found in the documentation. I will remove that

File: src/components/Navbar/navbar.module.scss (hunk 1 above)
Comment (Samson303) [reply src/components/Navbar/navbar.module.scss]: Thanks for the feedback

**Review by MathisBullinger — DISMISSED** [review]

**Review by MathisBullinger — APPROVED** [review]

**Pull request merged**